Quadriga/src/main/webapp/WEB-INF/views/public/sitesnavbar.jsp

```diff
@@ -0,0 +1,5 @@
+<li ${currentPage == "sites" ? "class=\"active\"" : ""}> <a href="">Quadriga Network Projects</a></li>
```

Comment:
you use "currentPage" in two jsp pages that are both displayed at the same time with different values, but it can only have one value at a time. that doesn't make sense. If you don't understand what this is doing, come to me :)